From 63ed0a287189b2021fa19edec75615bdce1878f8 Mon Sep 17 00:00:00 2001
From: David Abdurachmanov <david.abdurachmanov@sifive.com>
Date: Mon, 13 Sep 2021 03:22:32 -0700
Subject: [PATCH 7/7] riscv: sifive: unmatched: disable FDT and initrd
 relocation

Same as on SiFive Unleashed we need to disable fdt and initrd relocation. Tom
Rini mentined 18 days ago that it's most likely due to RISC-V lacking
`arch_lmb_reserve` implementation.

The patch seems to be submitted now:
[PATCH 09/12] lmb: riscv: Add arch_lmb_reserve()
https://lists.denx.de/pipermail/u-boot/2021-September/460333.html

Signed-off-by: David Abdurachmanov <david.abdurachmanov@sifive.com>
---
 include/configs/sifive-unmatched.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/configs/sifive-unmatched.h b/include/configs/sifive-unmatched.h
index d8eed92487..654a5db918 100644
--- a/include/configs/sifive-unmatched.h
+++ b/include/configs/sifive-unmatched.h
@@ -65,6 +65,8 @@
 	"name=system,size=-,bootable,type=${type_guid_gpt_system};"
 
 #define CONFIG_EXTRA_ENV_SETTINGS \
+	"fdt_high=0xffffffffffffffff\0" \
+	"initrd_high=0xffffffffffffffff\0" \
 	"kernel_addr_r=0x84000000\0" \
 	"fdt_addr_r=0x88000000\0" \
 	"scriptaddr=0x88100000\0" \
-- 
2.27.0

